cmd/observe/flows.go

```diff
@@ -293,13 +293,13 @@ more.`,
 
 	filterFlags.Var(filterVar(
 		"from-ip", ofilter,
-		"Show all flows originating at the given IP address."))
+		"Show all flows originating at the given IP address. Each of the source IPs can be specified as an exact match (e.g. '1.1.1.1') or as a CIDR range (e.g.'1.1.1.0/24')."))
 	filterFlags.Var(filterVar(
 		"ip", ofilter,
 		"Show all flows related to the given IP address."))
```

Comment:
Sorry, forgot that last time around: Could also add the sentence here, since it also applies to this? e.g. `Each of the IPs can be specified as an exact match (e.g. '1.1.1.1') or as a CIDR range (e.g.'1.1.1.0/24').`

Reply:
Sure